### PR TITLE
[7997446] [7997449] [7997527] Control changes from ADO Scanner docs

### PR DESCRIPTION
**Pull request by juhi037**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1 [hunk 1]
```diff
@@ -534,6 +535,93 @@ class Project: ADOSVTBase
...
+        }
+        if($this.BAMembers.Count -gt 0)
+        {
+            $TotalBAMembers = $this.BAMembers.Count
+            $controlResult.AddMessage("There are a total of $TotalBAMembers Build Administrators in your project.")
+            $controlResult.SetStateData("Count of Build Administrators: ",$TotalBAMembers)
+        }
+
+
+        if ($TotalBAMembers -gt 0)
```
Comment (arvindsingh-cloud) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1:552]: Already has if at line 544,  if($this.BAMembers.Count -gt 0) cant we add this if code udeer same

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1 [hunk 2]
```diff
@@ -534,6 +535,93 @@ class Project: ADOSVTBase
...
+                $svcAccountsCount = $svcAccounts.Count
+                #In case of graph access we will only evaluate the control on the basis of human accounts
+                if($humanAccountsCount -gt $this.ControlSettings.Project.MaxBAMembersPermissible){
+                    $controlResult.AddMessage([VerificationResult]::Failed,"Number of human build administrators configured are more than the approved limit: $($this.ControlSettings.Project.MaxBAMembersPermissible)");
+                }
+                else{
+                    $controlResult.AddMessage([VerificationResult]::Passed,"Number of human build administrators configured are within than the approved limit: $($this.ControlSettings.Project.MaxBAMembersPermissible)");
+                }
+
+                if($TotalBAMembers -gt 0){
```
Comment (arvindsingh-cloud) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1:570]: as this condition has already checked if($TotalBAMembers -gt 0) does it needed again?

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1 [hunk 3]
```diff
@@ -534,6 +535,93 @@ class Project: ADOSVTBase
...
+                $controlResult.AddMessage([Constants]::graphWarningMessage+"`n");
+                $this.BAMembers = @($this.BAMembers | Select-Object displayName,mailAddress)
+                if($TotalBAMembers -gt $this.ControlSettings.Project.MaxBAMembersPermissible){
+                    $controlResult.AddMessage([VerificationResult]::Failed,"Number of build administrators configured are more than the approved limit: $($this.ControlSettings.Project.MaxBAMembersPermissible).");
+                }
+                else{
+                    $controlResult.AddMessage([VerificationResult]::Passed,"Number of build administrators configured are within than the approved limit: $($this.ControlSettings.Project.MaxBAMembersPermissible).");
+                }
+
+                if($TotalBAMembers -gt 0){
```
Comment (arvindsingh-cloud) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1:604]: again same condition checking

File: src/AzSK.ADO/Framework/Configurations/SVT/ADO/ADO.Organization.json [hunk 4]
```diff
@@ -588,6 +588,24 @@
...
+      "Automated": "Yes",
+      "MethodName": "CheckRequestAccessPolicy",
+      "Rationale": "Access to the ADO instance should be allowed by only joining standard groups set up the respective teams while onboarding. Enabling this setting can allow a malicious user to access the instance and laterally move around groups, bypassing the designed security model.",
+      "Recommendation": "1. Go to Organization Settings --> 2. Security --> 3. Policies --> 4. Under 'User Policies' disable 'Request Access'",
+      "Tags": [
+          "SDL",
+          "TCP",
+          "Automated",
+          "AuthZ",
+          "AutomatedFix"
```
Comment (arvindsingh-cloud) [inline src/AzSK.ADO/Framework/Configurations/SVT/ADO/ADO.Organization.json:606]: AutomatedFix plz verify

File: src/AzSK.ADO/Framework/Helpers/AdministratorHelper.ps1 [hunk 5]
```diff
@@ -3,6 +3,7 @@ class AdministratorHelper{
     static [bool] $isCurrentUserPA=$false;
     static $AllPCAMembers = @()
     static $AllPAMembers = @()
+    static $AllBAMembers = @()
```
Comment (arvindsingh-cloud) [inline src/AzSK.ADO/Framework/Helpers/AdministratorHelper.ps1:6]: does it not needed clearing code?

Comment (juhi037) [reply src/AzSK.ADO/Framework/Helpers/AdministratorHelper.ps1:6]: already being cleared in GetTotalBAMembers method

**Review by arvindsingh-cloud — APPROVED** [review]

**Review by SaikumarRagala — APPROVED** [review]

**Pull request merged**